src/coreComponents/functions/TableFunction.hpp

```diff
@@ -317,11 +320,13 @@ class TableFunction : public FunctionBase
     m_valueUnit = unit;
   }
 
-  /**
-   * @brief Print table into a CSV file (only 1d and 2d tables are supported)
-   * @param filename Filename for output
-   */
-  void print( std::string const & filename ) const;
+/**
+ * @return The table unit
+ */
+  units::Unit getValueUnit() const
+  {
+    return m_valueUnit;
+  }
```

Comment:
When you have simple one-line setter/getter, you can do this way:
```suggestion
  { return m_valueUnit; }
```
Do as you prefer.